web/pandas/config.yml

```diff
@@ -139,7 +139,7 @@ workgroups:
     contact: communications@pandas.pydata.org
     responsibilities: "Share relevant information with the broader community, mainly via our social networks, as well as being the main point of contact between NumFOCUS and the core team."
     members:
-    - Marco Gorelli
+    - datapythonista
```

Comment:
wasn't sure what to put here, as removing myself would leave the team empty - @datapythonista is it ok if I put you here?